assignments/week_1/exercise_1_2/exercise_1_2.cpp

```diff
@@ -0,0 +1,34 @@
+// Iterative C++ program to reverse an array and check if string is palindrome
+#include <stdio.h>
+#include <string.h>
```

Comment:
Why do we need these two C headers for this exercise?

Reply:
Removed